client/app/views/templates/view_steps_confirmation.jade

```diff
@@ -5,7 +5,10 @@ block header
     h1=t('step confirmation title')
 
 block region
-    p=t('step confirmation description')
+    p
+        span= t('step confirmation description line 1')
+        br
```

Comment:
I think we should avoid using `<br/>` and prefer `<p>`, even if it does not match exactly the mockups. I asked @kelukelu and he agrees with this point of view.

Have a look to accounts step.